README.rst

```diff
@@ -15,7 +15,7 @@
 .. _Codecov: https://codecov.io/gh/scikit-learn/scikit-learn
 
 .. |Nightly wheels| image:: https://github.com/scikit-learn/scikit-learn/workflows/Wheel%20builder/badge.svg?event=schedule
-.. _`Nightly wheels`: https://github.com/scikit-learn/scikit-learn/actions?query=workflow%3A%22Wheel+builder%22+event%3Aschedule
+.. _` Nightly wheels`: https://github.com/scikit-learn/scikit-learn/actions?query=workflow%3A%22Wheel+builder%22+event%3Aschedule
```

Comment:
This change is not needed. There is an underscore before the backtick that makes this a link. You can test that link already works by going to the [rendered readme](https://github.com/scikit-learn/scikit-learn/blob/main/README.rst) and clicking on the "Wheel builder" badge. 

Can you revert this change?